src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+$card-count: 4;
+$row-gap: 50px;
+$padding: 41px;
+
+.catalog {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  max-width: $card-count * $card-width
+    + $padding * 2
+    + $row-gap * ($card-count - 1);
+  margin: 0 auto;
+  padding: 50px $padding;
+
```

Comment:
```suggestion
  max-width: $card-count * $card-width   
   
    + $row-gap * ($card-count - 1);
  margin: 0 auto;
  padding: 50px 0;

```

Reply:
your catalog is being centered anyway, maybe you don't need side paddings and formula with paddings can influense the tests